### DISCUSSION TITLE
Quasar Guide

### DISCUSSION THREAD

**Discussion by ha6755ad** [marshallswain/feathers-pinia#135] [General]
Hi, I have a working quasar implementation and am happy to share - it looks like you may also have already been working on this (or someone has) but if not I do have one.

**Comment by Theorian** [comment]
I have had a horrible experience using feathers-pinia with quasar, it is complicated, it is a mess and some of the strangest problems occur. I have wasted countless hours trying to troubleshoot strange absurdities. I do NOT recommend using this package with quasar, I would possibly recommend not even using this package for any app and just manage pinia and feathers data yourself. 

Auth was working, I am using the Quasar example EXACTLY. Then when I compile the app for capacitor on ios OR android, the user object is NOT populated after login, but is populated when using "quasar dev" after login.
